cmd/cloud/utils.go

```diff
@@ -0,0 +1,143 @@
+package cloud
```

Comment:
I think this entire code should do to the `internal/auth` package because its related to `auth` and may not be bounded only to the `cloud` command. May be used elsewhere in future.

From a domain perspective, this seems like an operation within the `auth` domain. So I feel its better to put this code in the `auth` package